data/templates/sangoma.tmpl

```diff
@@ -205,10 +205,10 @@
         <P20165 para="FirmwareUpGrade.PnPActive">0</P20165>
         <P20172 para="Preference.VmTransfer">0</P20172>
         <P20173 para="Preference.VmTransferCode" />
-        <P20174 para="FirmwareUpGrade.RingUrl">{{ ringtone_url }}</P20174>
+        <P20174 para="FirmwareUpGrade.RingUrl">{{ ringtone_file ? "#{provisioning_url_scheme}://#{hostname}/#{provisioning_url_path}#{tok2}/ringtone/#{ringtone_file}?" : "" }}</P20174>
```

Comment:
I'd use "ringtones" here in the path for coherence with the api endpoint. Also provisioning endpoint is missing, but I guess that this is my fault 

```suggestion
        <P20174 para="FirmwareUpGrade.RingUrl">{{ ringtone_file ? "#{provisioning_url_scheme}://#{hostname}/#{provisioning_url_path}#{tok2}/ringtones/#{ringtone_file}?" : "" }}</P20174>
```

Reply:
I added it here https://github.com/nethesis/tancredi/pull/157